### PR TITLE
Added support for tips with attributed text

### PR DESCRIPTION
**Pull request by lawmaestro**
* Added unit tests for `TipSeeManager`.
  * Added attributed text tip example to `TipSee-Example` project.
  * Updated file comment header blocks to be consistent.
  * Removed superfluous imports.
  * Cleaned up minor formatting inconsistencies.
  * Renaming of a few files to correctly reflect the contained class.
  * Removed `.DS_Store` from version control.
  * Removed unused podspec.

### CODE REVIEW

**Comment by fredjahanmanesh** [comment]
thank you i will check it ASAP

**Comment by lawmaestro** [comment]
> thank you i will check it ASAP

Hey, just wondered if you'd had chance to take a look over this yet?

**Comment by fredjahanmanesh** [comment]
I’m sorry about this delay, we are in the middle of new release in the company and my time is so tight because we work from morning to midnight, but i promise that i would take a look at this in the weekend

File: TipSee/Classes/NSAttributedString+Size.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        let rect = self.boundingRect(with: CGSize(width: widthConstraint, height: CGFloat.greatestFiniteMagnitude),
+									 options: [.usesLineFragmentOrigin, .usesFontLeading],
+									 context: nil)
+        return ceil(rect.size.height)
+	}
+	func width(widthConstraint: CGFloat, heightConstraint: CGFloat) -> CGFloat {
+		let rect = self.boundingRect(with: CGSize(width: widthConstraint, height: heightConstraint),
+									 options: [.usesLineFragmentOrigin, .usesFontLeading],
+									 context: nil)
+        return ceil(rect.size.width + 10)
```
Comment (fredjahanmanesh) [inline TipSee/Classes/NSAttributedString+Size.swift:19]: why do we add a static `10` to the width?

**Review by fredjahanmanesh — APPROVED** [review]
well done Adam

**Pull request merged**